### PR TITLE
BACKLOG-12965 : fix the sizes

### PR DESCRIPTION
**Pull request by Yousria**
## JIRA

https://jira.jahia.org/browse/BACKLOG-12965

## Tests

The following are included in this PR
- [ ] Unit Tests (Most changes _should_ have unit tests)
- [ ] Integration Tests

## Checklist

I have considered the following implications of my change: 

- [ ] Security (in particular for changes to authentication, authorization, data fetching, ...)
- [ ] Performance
- [ ] Migration
- [ ] Code maintainability

## Documentation

- [ ] Inline documentation
- [ ] Internal Documentation (wiki)
- [ ] User-facing Documentation

### CODE REVIEW

File: src/javascript/JContent/actions/publishAction.jsx [hunk 1]
```diff
@@ -69,9 +69,8 @@ export const PublishActionComponent = ({context, render: Render, loading: Loadin
         displayName: _.escape(ellipsizeText(res.node.displayName, 40)),
         language: res.node.site ? _.escape(uppercaseFirst(getLanguageLabel(res.node.site.languages, language).displayName)) : null
     } : {
-        displayName: t('jcontent:label.contentManager.selection.itemsSelected', {count: context.paths.length}),
-        language: res.nodes[0].site ? _.escape(uppercaseFirst(getLanguageLabel(res.nodes[0].site.languages, language).displayName)) : null,
-        multipleSelectionDisplayName: t('jcontent:label.contentManager.selection.items', {count: context.paths.length})
+        displayName: t('jcontent:label.contentManager.selection.items', {count: context.paths.length}),
```
Comment (tdraier) [inline src/javascript/JContent/actions/publishAction.jsx]: you still have some stuff in getButtonRenderer that can be cleaned

File: src/javascript/JContent/ContentRoute/ToolBar/ToolBar.jsx [hunk 2]
```diff
@@ -31,22 +31,24 @@ export const ToolBar = () => {
...
                     <Separator variant="vertical" invisible="onlyChild"/>
                     <DisplayActions
                         target="selectedContentActions"
                         context={{paths: selection}}
                         filter={action => action.key !== 'deletePermanently' && action.key.indexOf('publish') === -1}
-                        render={getButtonRenderer({size: 'small', variant: 'ghost'})}
+                        render={getButtonRenderer({size: 'default', variant: 'ghost'})}
                     />
                 </div>
                 <Separator variant="vertical" invisible="onlyChild"/>
```
Comment (tdraier) [inline src/javascript/JContent/ContentRoute/ToolBar/ToolBar.jsx]: You don't have the "edit" button when selection is on one item ? Is it still required ? (still there in the mockups)

Comment (Yousria) [reply src/javascript/JContent/ContentRoute/ToolBar/ToolBar.jsx]: we had that in cmm before? As far as I know 'edit' is not part of multiple selection even if it's 1 item, maybe I'm wrong

File: src/javascript/JContent/actions/publishAction.jsx (hunk 1 above)
Comment (Yousria) [reply src/javascript/JContent/actions/publishAction.jsx]: yep

File: src/javascript/JContent/ContentRoute/ToolBar/ToolBar.jsx [hunk 3]
```diff
@@ -31,23 +31,25 @@ export const ToolBar = () => {
...
-                    {t('jcontent:label.contentManager.selection.itemsSelected', {count: selection.length})}
-                </Typography>
-                <div className={styles.spacer}/>
-                <Button icon={<Cancel/>} variant="ghost" size="small" onClick={() => dispatch(cmClearSelection())}/>
+                <div className={`flexRow ${styles.selection}`}>
+                    <Typography variant="caption" data-cm-role="selection-infos" data-cm-selection-size={selection.length}>
+                        {t('jcontent:label.contentManager.selection.itemsSelected', {count: selection.length})}
+                    </Typography>
+                    <div className={styles.spacer}/>
+                    <Button icon={<Cancel/>} variant="ghost" size="small" onClick={() => dispatch(cmClearSelection())}/>
```
Comment (cedricmoitrier) [inline src/javascript/JContent/ContentRoute/ToolBar/ToolBar.jsx]: Looks like you forgot to update this button size ('small' while others are using 'default')

File: src/javascript/JContent/ContentRoute/ToolBar/Toolbar.scss [hunk 4]
```diff
@@ -5,3 +5,7 @@
 .spacer {
     width: var(--spacing-small);
 }
+
+.selection {
+    margin-top: 5px;
```
Comment (cedricmoitrier) [inline src/javascript/JContent/ContentRoute/ToolBar/Toolbar.scss]: You should use moonstone's tokens...

Comment (cedricmoitrier) [reply src/javascript/JContent/ContentRoute/ToolBar/Toolbar.scss]: But I am also wondering if this is not a dirty hack.... :confused:
The difference of heights is likely related to buttons/texts using different sizes, doesn't it?

File: src/javascript/JContent/ContentRoute/ToolBar/ToolBar.jsx (hunk 3 above)
Comment (tdraier) [reply src/javascript/JContent/ContentRoute/ToolBar/ToolBar.jsx]: I was also wondering, but looking at the mockups it seems to be smaller than the others indeed ...

File: src/javascript/JContent/ContentRoute/ToolBar/ToolBar.jsx (hunk 2 above)
Comment (tdraier) [reply src/javascript/JContent/ContentRoute/ToolBar/ToolBar.jsx]: No, never had that in CMM. But it's part of jcontent mockup ? Anyway, that will require changes in content editor

**Pull request merged**